package exceptionchaining;

import java.util.Scanner;

public class Demo {

	public static void main(String[] args) throws PojarException {
		House kushta = new House();
		kushta.p = new Person();
		kushta.throwParty();
		
		
		
		try {
			Scanner scanner = new Scanner(System.in);
			
					} catch (Exception e) {
			// TODO: handle exception
		}
	}

}
